features/step_definitions/series_steps.rb

```diff
@@ -60,3 +60,7 @@
     click_button("Post Without Preview")
   end
 end
+
+Then /^the series "(.*)" should be deleted/ do |series|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.